client/my-sites/plans-comparison/use-plan-prices.tsx

```diff
@@ -31,6 +31,9 @@ function toMonthlyPrice( plan: WPComPlan ) {
 	};
 }
 
+/**
+ * @deprecated
+ */
```

Comment:
This hook has the same name as the one added recently. It seems to be having similar functionality, and I found only 1 usage. @chriskmnds, wondering if you had noticed this one and if it can be replaced by the new hook? For now, I added the `@deprecated` comment so that it is not used by mistake.